DependencyInjection/SmartboxIntegrationFrameworkExtension.php

```diff
@@ -108,6 +109,11 @@ public function loadConfigurableProducers(ContainerBuilder $container)
             $definition->addMethodCall('setSerializer', [new Reference('jms_serializer')]);
             $definition->addMethodCall('setEventDispatcher', [new Reference('event_dispatcher')]);
             $definition->addMethodCall('setName', [$producerName]);
+
+            if(is_subclass_of($class, RestConfigurableClientInterface::class)) {
```

Comment:
Can't be use here is **instanceof**?

Reply:
We can if we use an instance (new $class) instanceOf to compare objects.
Comparing string with FCQN does not seems to work properly

`
$interfaceName = RestConfigurableClientInterface::class;

            if(is_subclass_of($class, $interfaceName)) //true
         
            if($class instanceof $interfaceName) //false`

Reply:
With the instance of you need to specify the class itself, not the class name, maybe for this reason doesn't work well

Reply:
Thats just what I said at the first statement.
if we use an instance (new $class) instanceOf to compare objects.
Or is an object comparison or the usage of subclass
$class (its a FCQN) instanceOf Interface (is an object) does not work

Reply:
Thanks for the clarification :)

As bonus track, can you run phpcs on the changes you did?

Reply:
Done!